examples/pytorch/bert/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
Let's remove entire `bert` example here. I think Facebook adds an example https://github.com/kubeflow/pytorch-operator/pull/345 here. We may not carefully look at the licenses. 

To avoid potential license issue, it would be great to remove them.